docs/oauth.md

```diff
@@ -0,0 +1,231 @@
+---
+title: OAuth Authentication Support
+---
+**IMPORTANT:** This feature is available in Tableau 2021.1 and newer. 
+
+# How to enable OAuth for a plugin connector
+
+First check your database and driver documentation to make sure it supports OAuth. For a complete example please refer to https://github.com/tableau/connector-plugin-sdk/tree/dev-2021.1/samples/scenarios/snowflake_oauth.
```

Comment:
Link to master

Reply:
Updated the links to get samples from master